shift_detector/utils/column_management.py

```diff
@@ -58,10 +65,13 @@ def detect_column_types(df1, df2, columns):
         ...
     }
     """
-    numerical_columns = [c for c in columns if is_numeric_dtype(df1[c]) and is_numeric_dtype(df2[c])]
+    categorical_columns = [c for c in columns if is_binary(df1[c], True) and is_binary(df2[c], True)]
```

Comment:
unless the data contains less than 20 values, binary cols should be recognized as categorical anyway, because the default distinctness threshold is 0.1. If both true and false occur, there are two distinct values and therefore with 20 rows or more this is always fulfilled. If nan is counted there are 3 values and so at least 30 values are required

Reply:
what I'm trying to say is: do we really need this distinction for the rare case less than 30 rows are passed?

Reply:
It is necessary. Otherwise the boolean column will be detected as numercial column.